packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -86,11 +86,13 @@ const AccountWizard = observer(props => {
         sub_section_index: ui.sub_section_index,
     };
 
+    const { temp_formik_data, setTempFormikData, temp_formik_data_step, setTempFormikDataStep } = client;
```

Comment:
Good thing in saving the state. Can we rename it to something more descriptive with real_account_form_date or anything. Coz temp doesn't fit the context here IMO

Reply:
@shahzaib-deriv agree, will change it soon